.circleci/config.yml

```diff
@@ -76,7 +76,7 @@ jobs:
   OPENCL:
     environment:
       DOCKER_IMAGE: "308535385114.dkr.ecr.us-east-1.amazonaws.com/caffe2/py2-clang8-ubuntu16.04:283"
-    <<: *linux_default 
```

Comment:
nit this change seems unnecessary

Reply:
hmmmmmmm not so sure what have been changed here...